src/main/java/net/atos/client/bag/api/AdresUitgebreidApi.java

```diff
@@ -55,34 +81,50 @@
+     * De volgende (combinaties van) parameters worden ondersteund: 1. Bevragen van de uitgebreide informatie van één of meer huidige
+     * adressen met een postcode, huisnummer en optioneel huisnummertoevoeging en huisletter. Het opgeven van een combinatie van parameters
+     * levert niet altijd een exacte match met één adres, bv. in geval van meerdere objecten met huisnummertoevoegingen en/of huisletters.
+     * Met de exacteMatch parameter kan worden aangegeven dat alleen object(en) die exact overeenkomen met de opgegeven parameters,
+     * geretourneerd moeten worden. 2. Bevragen van de uitgebreide informatie van één of meer huidige adressen met de identificatie van een
+     * adresseerbaar object. 3. Bevragen van de uitgebreide informatie van één of meer huidige adressen met woonplaats naam, openbare ruimte
+     * naam, huisnummer en optioneel huisnummertoevoeging en huisletter. Het opgeven van een combinatie van parameters levert niet altijd
+     * een exacte match met één adres, bv. in geval van meerdere objecten met huisnummertoevoegingen en/of huisletters. Met de exacteMatch
+     * parameter kan worden aangegeven dat alleen object(en) die exact overeenkomen met de opgegeven parameters, geretourneerd moeten
+     * worden. 4. Zoek uitgebreide adres informatie van huidige adressen met een zoekterm. Voor paginering, zie: [functionele specificatie
+     * paginering](https://github.com/lvbag/BAG-API/blob/master/Features/paginering.feature). Als inclusiefEindStatus&#x3D;true, dan worden
+     * ook actuele adressen met een eind status geleverd, zie [functionele specificatie
+     * inclusiefEindstatus](https://github.com/lvbag/BAG-API/blob/master/Features/inclusief-eind-status.feature).
      */
     @GET
     @Produces({"application/hal+json", "application/problem+json"})
     public AdresUitgebreidHalCollection zoekAdresUitgebreid(@QueryParam("postcode") String postcode,
```

Comment:
I see it doesn't automatically move this `@QueryParam("postcode") String postcode,` onto the next line. A shame. That is what e.g. Detekt does for Kotlin and something I like much better. Guess we still have some manual work left then to improve the formatting.

Reply:
Changed